community/gallery/collection/03_programs.mdx

```diff
@@ -587,6 +587,19 @@ zi light-mode for id-as'pnpm' from'gh-r' bpick'*-linux-x64' as'program' \
     pnpm/pnpm
 ```
 
+### GH-R: [volta-cli/volta][https://github.com/volta-cli/volta]
+
+```shell showLineNumbers
+zi light cowboyd/zsh-volta
+# completions: 
+# OMZP::volta uses ZSH_CACHE_DIR to search for the completions directory
+# usually completions are located at $HOME/.zi/completions
+zi wait'0b' lucid for \
+  blockf has'volta' \
+  atinit'ZSH_CACHE_DIR=$ZI[HOME_DIR]' \
```

Comment:
@reviewer: `ZI[CACHE_DIR]` did not work (on my system), because it did not exist and therefore the completions plugin could not find a completions directory under this location.

Reply:
> My question is, does this work for other users as well? Where are their completions located?

This is a good question. 

It's only 14 lines of code for simple tasks and is done poorly.

1. The [volta completion plugin](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/volta/volta.plugin.zsh) actually is function.
2. Variable `ZSH_CACHE_DIR` shouldn't be hardcoded in this situation.
3. It should [follow standards](https://wiki.zshell.dev/community/zsh_plugin_standard) or be reasonable.

We can't include such a plugin in our wiki as an example. I should have a solution for this in the upcoming weekend.